libs/wingsdk/src/simulator/simulator.ts

```diff
@@ -643,30 +681,19 @@ export class Simulator {
    */
   private tryResolveTokens(obj: any): any {
     if (typeof obj === "string") {
-      if (isToken(obj)) {
-        const ref = obj.slice(2, -1);
-        const [path, rest] = ref.split("#");
-        const config = this.getResourceConfig(path);
-        if (rest.startsWith("attrs.")) {
-          const attrName = rest.slice(6);
-          const attr = config?.attrs[attrName];
-
-          // we couldn't find the attribute. this doesn't mean it doesn't exist, it's just likely
-          // that this resource haven't been started yet. so return `undefined`, which will cause
-          // this resource to go back to the init queue.
-          if (!attr) {
+      let resolveCount = 0; // prevent infinite loop, just in case
+      while (resolveCount < 100) {
```

Comment:
Does each token have 100 tries to be resolved? that are executed one after the other?

Reply:
I updated this to avoid this while loop hack